backend/psql/migrations/011-media.sql

```diff
@@ -0,0 +1,23 @@
+-- +migrate Up
+
+-- new tables for media
+
+CREATE TABLE media {
+	media_id TEXT NOT NULL PRIMARY KEY,
+	room TEXT NOT NULL,
+	storage TEXT NOT NULL,
+	uploader TEXT NOT NULL,
+	created TIMESTAMP with time zone NOT NULL,
+	uploaded TIMESTAMP with time zone
+};
+
```

Comment:
We should go ahead and put some indices in place:

* room, created
* uploader, created
* room, uploader, created